operations/system_management_health/Troubleshoot_Grafana_Dashboard.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshoot Grafana Dashboard
+
+General Grafana Dashboard Troubleshooting Topics
+- [Troubleshoot Grafana dashboard](#troubleshoot-grafana-dashboard)
+  - [Ceph - OSD Overview Dashboard](#ceph-osd-overview-dashboard)
+  - [Ceph - RBD Overview Dashboard](#ceph-rbd-overview-dashboard)
+  - [Ceph - RGW Instance Detail Dashboard](#ceph-rgw-instance-detail-dashboard)
+  - [Ceph - RGW Overview Dashboard](#ceph-rgw-overview-dashboard)
+
+<a name="ceph-osd-overview-dashboard"></a>
+## Ceph - OSD Overview Dashboard: 3 panels not found
+
+Currently skipping cray-sysmgmt-health pie chart plugin installation for airgaped systems. If the system is non airgaped then comment out or remove plugins property in customizations.yaml.
```

Comment:
- Can we show an example codeblock here of what lines an admin will need to comment out or remove?
- Also, if we are suggesting editing the customizations.yaml file, it might be a good idea to provide the commands to reupload it to Kubernetes so the changes persist.
  ```
  ncn-m001# kubectl delete secret -n loftsman site-init
  ncn-m001# kubectl create secret -n loftsman generic site-init --from-file=customizations.yaml 
  ```